libs/openchallenges/team/src/lib/team.component.html

```diff
@@ -15,7 +15,7 @@ <h2>Meet Our Team</h2>
     <div class="member-card">
       <img class="member-pic" [src]="(thomasAvatar$ | async)?.url" alt="Thomas Schaffter" />
       <h3 class="card-name">Thomas Schaffter</h3>
-      <p class="card-roles">Product owner, Architect, Lead full-stack engineer</p>
+      <p class="card-roles">OC Team Lead</p>
```

Comment:
The title request for Gaia and I was "Team Lead" and "Data Lead". Here we are in the context of OC and so there is no need to add the OC prefix.
```suggestion
      <p class="card-roles">Team Lead</p>
```